source/blog/tag.html.erb

```diff
@@ -1,3 +1,4 @@
+<% add_atom_feed_to_head %>
```

Comment:
I'm uncertain whether this should exist on scoped pages (e.g. this tag page, the author page) as it's offering the 'unfiltered' blog as the alternative view of this page, from the source:
```HTML
<link rel="alternate" type="application/atom+xml" title="The Unboxed Blog" href="/blog/feed.xml" />
```
is it normal to offer an atom alternative of the unfiltered index page across a blog, regardless of what article/filtered view the user is on?